src/ontology/uberon-edit.obo

```diff
@@ -222010,6 +222010,15 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:0003687 ! foramen magnum
 property_value: depicted_by http://locatelambda.org/wp-content/uploads/2018/02/opisthion.jpg xsd:string
 
+[Term]
+id: UBERON:7500119
+name: ectorbitale
+def: "An anatomical point on the most lateral point of the frontal bone on the occipital side of the orbit." [ISBN:9004086161, ISBN:9789004086166]
```

Comment:
@megbalk do you think you should mention this anatomical entity is in tetrapods only in the text def?

Reply:
I can do that!